exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/MetricAdapter.java

```diff
@@ -328,5 +331,24 @@ private static ByteString convertSpanId(String id) {
         OtelEncodingUtils.bytesFromBase16(id, SpanId.getLength()));
   }
 
+  // Fill labels too until Collector supports attributes and users have had a chance to update.
+  @SuppressWarnings("deprecation")
+  private static void fillAttributes(
+      Attributes attributes,
+      Consumer<KeyValue> attributeSetter,
+      Consumer<io.opentelemetry.proto.common.v1.StringKeyValue> labelSetter) {
+    attributes.forEach(
+        (key, value) -> {
+          attributeSetter.accept(CommonAdapter.toProtoAttribute(key, value));
+          if (key.getType() == AttributeType.STRING) {
```

Comment:
Nit: You should be `.toString()` some of the other primitive types (like int, double, boolean).

Reply:
Ok went with toString without checking type